reactive-datatable-template.js

```diff
@@ -23,7 +23,31 @@ Template.ReactiveDatatable.rendered = function() {
         reactiveDataTable.page = info.page;
     });
 
+    // Data difference
+    reactiveDataTable.difference = function(existingData, newData) {
```

Comment:
looks good, but why is this in the template and not in with the rest of the logic?